ibm_mq/README.md

```diff
@@ -158,6 +158,8 @@ No commands have a syntax error.
 All valid MQSC commands were processed.
 ```
 
+**Note**: The Datadog Agent will attempt to collect metrics for [reset queue statistics][14] (`MQCMD_RESET_Q_STATS`) by default. These metrics will require granting further `+chg` permissions to the `datadog` user in order to collect these metrics. Collecting reset queue statistics performance data will also **reset** the performance data.
```

Comment:
I think this should be moved to the top of this section and make it something like this:

There are many ways to set up permissions in IBM MQ. Depending on how your setup works, create a `datadog` user within MQ with read only permissions and, optionally, `+chg` permissions. `+chg` permissions are required to collect metrics for [reset queue statistics][14] (`MQCMD_RESET_Q_STATS`). If you do not wish to collect these metrics you can disable `collect_reset_queue_metrics` on the configuration.

**Note**: Collecting reset queue statistics performance data will also **reset** the performance data.